### ISSUE TITLE
getAuthor error on assets

### ISSUE THREAD

**Issue by billymedia** [verbb/comments#230]
When trying to leave a comment in a front end comment form against an asset uploaded in a front end entry form, i get an error about the author. The asset does have an author (uploaded by).

2021-06-10 12:49:02 [-][2][-][info][comments] Prepare Author Notifications.
2021-06-10 12:49:02 [-][2][-][info][comments] Not sending element author notification, no author found: Calling unknown method: craft\elements\Asset::getAuthor()
2021-06-10 12:49:02 [-][2][-][info][comments] Cannot send element author notification: No recipient null

**Steps to reproduce**
1. Leave a comment against an asset

**Additional info**
- Plugin version: 1.8.7
- Craft version: 3.6.16
- Multi-site: No

**Comment by engram-design** [comment]
So technically an Asset element doesn't have an author, but has an "Uploader" which was introduced in Craft 3.4.0. Comments specifically looks for `$element->getAuthor()` which rightly errors here, as that method doesn't exist on an asset.

Fixed for the next release. To get the fix early, change your `verbb/comments` requirement in `composer.json` to:

```
"require": {
  "verbb/comments": "dev-craft-3 as 1.8.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by billymedia** [comment]
Perfect, that works fine now.

**Comment by engram-design** [comment]
Fixed in [1.8.8](https://github.com/verbb/comments/releases/tag/1.8.8)
